src/main/resources/db/migration/V2__alter_dialogmeldingdokument.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE DIALOGMELDINGDOKUMENT
+ADD COLUMN sha_string VARCHAR(64) UNIQUE NOT NULL DEFAULT '';
```

Comment:
Kan slenge på en indeks for `sha_string ` om det er den vi kommer til å gjøre oppslag mot i stedet for id i `erDialogmeldingOpplysningerLagret`

Reply:
Tror Postgres automatisk lager indeks når det er en UNIQUE-constraint, men se kommentaren min nedenfor.